source/classes.tex

```diff
@@ -866,7 +866,8 @@
 An inline member function (whether static or non-static) may
 also be defined outside of its class definition provided either its
 declaration in the class definition or its definition outside of the
-class definition declares the function as \tcode{inline} or \tcode{constexpr}.
+class definition declares the function as
+\tcode{inline}, \tcode{constexpr}, or \tcode{consteval}.
```

Comment:
I was going to say: maybe just "declares the function as an inline function".

But then this sentence says: "an inline member function may be defined outside the class if it's an inline function". Which reduces to "an inline member function may be defined outside the class". But that's also true for non-inline member functions and we said it in the previous paragraph. So what point are we trying to make here exactly?

Maybe this should be a note attached to the previous paragraph? Or removed entirely?

Reply:
I've fixed this with a slightly larger hammer.